src/fu-plugin.c

```diff
@@ -529,6 +529,43 @@ fu_plugin_get_hwids (FuPlugin *self)
 	return fu_hwids_get_guids (priv->hwids);
 }
 
+/**
+ * fu_plugin_has_custom_flag:
+ * @self: A #FuPlugin
+ * @flag: A custom text flag, specific to the plugin, e.g. `uefi-force-enable`
+ *
+ * Returns if a per-plugin HwId custom flag exists, typically added from a DMI quirk.
+ *
+ * Returns: %TRUE if the quirk entry exists
+ *
+ * Since: 1.3.1
+ **/
+gboolean
+fu_plugin_has_custom_flag (FuPlugin *self, const gchar *flag)
+{
+	FuPluginPrivate *priv = GET_PRIVATE (self);
```

Comment:
This can be moved into the for loop it looks like (and not running in the hwids == NULL case)